CODINGSTYLE.md

```diff
@@ -224,7 +224,7 @@ The purpose of a commit message is to summarize the scope and context of a patch
 
 ## Documentation
 
-1. Write your comments in a `typedoc` compliant way.
+1. Write your comments in a [TSDoc](https://github.com/Microsoft/tsdoc) compliant way.
```

Comment:
Minor, you can specify the actual URL at the end of the page, see: https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#links

Reply:
Ahm that's pretty cool. Did not know that